src/containers/ProfilePage.jsx

```diff
@@ -78,6 +79,11 @@ class ProfilePage extends React.Component {
         })
         .then(res => res.json())
         .then((res) => {
+          if (res.links.length === 0 && !res._defaultprofile) {
+            this.setState({
+              permission: true,
```

Comment:
Can we rename `permission` to something clearer? Right now it looks like you *do* have permission to view this resource by setting `permission` to true.
Perhaps use `invalidPermission` or keep `permission` and set it to false when you don't have permission?